### PR TITLE
feat: dashboard for ERN Ithaca

### PR DESCRIPTION
**Pull request by davidruvolo51**
This request introduces the initial website for ERN Ithaca registry. This includes several landing pages and a public dashboard.

## Tasks

- [x] create new vue app from template
- [x] Create pages for each view using the latest changes in the `molgenis-viz` library. 
- [x] Rewrite the dashboard from scratch using the new dashboard layout components
- [x] Rewrite data retrieval steps using graphql
- [x] Rewrote data handling steps
- [ ] Review

## Creating the preview

1. commit changes and push to the branch `feat/ern-ithaca`
2. wait for the preview to build
3. when the preview is ready, create a new schema using the "Dashboard" template. Give it a name and do not load the example dataset.
4. Import the demo dataset from https://github.com/molgenis/projects-rd-erns/blob/main/erns/ithaca/ithaca_emx2.xlsx.
5. Navigate to `<schema>/ern-ithaca` to view the app

See https://preview-emx2-pr-2768.dev.molgenis.org/ERN%20Ithaca/ern-ithaca/#/

### CODE REVIEW

**Review by mswertz — APPROVED** [review]

File: apps/molgenis-viz/src/components/layouts/Dashboard.vue [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  padding: 2em 0;
+
+  .dashboard-content {
+    width: 90%;
+    margin: 0 auto;
+    padding: 2em 0;
+    gap: 2em;
+  }
+}
+</style>
```
Comment (chinook25) [inline apps/molgenis-viz/src/components/layouts/Dashboard.vue:21]: missing enter at the end (which the auto-formatter wants)

File: apps/molgenis-viz/src/components/layouts/DashboardBox.vue [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+</template>
+
+<style lang="scss">
+.dashboard-box {
+  box-shadow: $box-shadow;
+  box-sizing: content-box;
+  border-radius: 8px;
+  background-color: $gray-000;
+}
+</style>
```
Comment (chinook25) [inline apps/molgenis-viz/src/components/layouts/DashboardBox.vue:14]: missing enter at the end (which the auto-formatter wants)

File: apps/ern-ithaca/vite.config.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import { defineConfig } from "vite";
+import vue from "@vitejs/plugin-vue";
+
+const host = "https://ithaca-emx2test.molgeniscloud.org/";
```
Comment (chinook25) [inline apps/ern-ithaca/vite.config.js:4]: Something to discuss during a dev-meet I guess, but is it ok to let apps refer to test servers like this. I'd prefer to have it run on localhost so I can test things in isolation.

File: apps/ern-ithaca/src/components/AppFooter.vue [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+              >
+            </li>
+            <!-- more links here -->
+          </UnorderedList>
+        </div>
+        <div class="footer-column footer-links">
+          <p><strong>For Members</strong></p>
+          <UnorderedList listType="none">
+            <li>
+              <a href="/apps/central/#/admin/users">Sign in</a>
```
Comment (chinook25) [inline apps/ern-ithaca/src/components/AppFooter.vue:28]: Why is this linking to the admin user page? I'd expect this link to open the sign in modal, just like the sign in button at the top of the page.

**Review by chinook25 — CHANGES_REQUESTED** [review]
Mostly small nitpicks and some questions. 
One other thing I noticed, the footer at the bottom of the screen is missing a space: 
This database was created using [MOLGENIS open source software](https://www.molgenis.org/)using version ....
between the link and using.

File: apps/ern-ithaca/index.html [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+<body>
+<noscript>
+    <strong>We're sorry but hello-world app doesn't work properly without JavaScript enabled.
+        Please enable it to continue.</strong>
+</noscript>
+<div id="app">
+</div>
+<script type="module" src="/src/main.js"></script>
+</body>
+</html>
```
[the remark this answers is not in the document]
Comment (davidruvolo51) [reply apps/ern-ithaca/index.html:20]: ✅

File: apps/ern-ithaca/index.html [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="icon" href="/favicon.ico">
+    <link rel="stylesheet" href="theme.css">
+    <title>ERN CRANIO</title>
+</head>
+<body>
+<noscript>
+    <strong>We're sorry but hello-world app doesn't work properly without JavaScript enabled.
```
[the remark this answers is not in the document]
Comment (davidruvolo51) [reply apps/ern-ithaca/index.html:13]: ✅ (This was leftover from the template)

File: apps/ern-ithaca/src/views/view-documents.vue [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+      height="large"
+    />
+    <PageSection
+      id="section-temp-message"
+      aria-labelledby="section-temp-message"
+      :verticalPadding="2"
+    >
+      <h2 id="section-templ-message">Documents</h2>
+      <MessageBox type="warning">
+        <p>This page is under construction.</p>
```
[the remark this answers is not in the document]
Comment (davidruvolo51) [reply apps/ern-ithaca/src/views/view-documents.vue:16]: Yes, we do not yet received the corresponding text and links for this page.

File: apps/ern-ithaca/src/components/AppFooter.vue (hunk 4 above)
Comment (davidruvolo51) [reply apps/ern-ithaca/src/components/AppFooter.vue:28]: I'm removing this. I tried to implement this earlier, but it was decided that it wasn't necessary to have a sign in link in the footer.

File: apps/ern-ithaca/src/utils/utils.js [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+  const response = await fetch(url, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+      Accept: "application/json",
+    },
+    body: JSON.stringify({ query: query }),
+  });
+  return response.json();
+}
```
[the remark this answers is not in the document]
Comment (davidruvolo51) [reply apps/ern-ithaca/src/utils/utils.js:11]: I'm now using the request function

**Review by chinook25 — APPROVED** [review]

**Pull request merged**